cpp/src/groupby/groupby.cu

```diff
@@ -108,15 +110,17 @@ struct empty_column_constructor {
 
     if constexpr (k == aggregation::Kind::COLLECT_LIST || k == aggregation::Kind::COLLECT_SET) {
       return make_lists_column(
-        0, make_empty_column(type_to_id<size_type>()), empty_like(values), 0, {});
+        0, make_empty_column(type_to_id<size_type>()), empty_like(values), 0, {}, stream, mr);
     }
 
     if constexpr (k == aggregation::Kind::HISTOGRAM) {
       return make_lists_column(0,
```

Comment:
Could this just use `cudf::lists::detail::make_empty_lists_column` instead?
https://github.com/rapidsai/cudf/blob/dfb7c113b9a0e931d40a65b8f3e51ab0295c2284/cpp/include/cudf/lists/detail/lists_column_factories.hpp#L50

Reply:
Cannot do that, due to:
> C++ exception with description "CUDF failure at: ../../src/column/column_factories.cpp:60: make_empty_column is invalid to call on nested types" thrown in the test body.